extensions/test/CognitoAuthentication.IntegrationTests.Xamarin/CognitoAuthentication.IntegrationTests.Xamarin.Android/CognitoAuthentication.IntegrationTests.Xamarin.Android.csproj

```diff
@@ -0,0 +1,107 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are Android projects required to be inthe old csproj format?

Reply:
This is not necessary